lib/matplotlib/axis.pyi

```diff
@@ -57,9 +57,6 @@ class Tick(martist.Artist):
     def set_pad(self, val: float) -> None: ...
     def get_pad(self) -> None: ...
     def get_loc(self) -> float: ...
-    def set_label1(self, s: object) -> None: ...
-    def set_label(self, s: object) -> None: ...
-    def set_label2(self, s: object) -> None: ...
```

Comment:
Either need to stay for deprecation period or be added to the `ci/mypy-stubtest-allowlist.txt`

I'm largely okay with either, did more of the latter in initial passes, though much of what was ignored was slated to be removed for 3.8 anyway, so it was just a matter of expiring deprecations.

Decorator use (outside of builtins like `property` or `staticmethod` and the like) is generally discouraged/not guaranteed to work in stubs (I may need to take a look to see how well we follow that guidance), So the options are:

- Keep in stubs
  - reflects actual available API for the given version
  - perhaps encourages use of deprecated API, though
- Remove from stubs:
  - Requires silencing stubtest
  - gives users who have type checking enabled one more early warning sign that they should update their code, preventing people from missing it due to e.g. warnings being silenced in CI.

Reply:
I just left in the old ones.